documentation/src/main/asciidoc/user_guide/configuration.adoc

```diff
@@ -408,7 +460,7 @@ $ java -cp ... -Djgroups.tcp.port=1234 -Djgroups.tcp.address=10.11.12.13
 
 
 ==== Further reading
-JGroups also supports more system property overrides, details of which can be found on this page: link:http://community.jboss.org/docs/12352[SystemProps] 
+JGroups also supports more system property overrides, details of which can be found on this page: link:http://community.jboss.org/docs/12352[SystemProps]
```

Comment:
I've asked @belaban to find a better home for that and he has opened https://issues.jboss.org/browse/JGRP-2107
We will update this when it becomes available